tests/libyui/menu_hotkeys_1177760_spec.rb

```diff
@@ -0,0 +1,36 @@
+require_relative "rspec_tmux_tui"
+
+describe "Menu Item" do
+  bug = "1177760"
+  around(:each) do |ex|
+    @base = "menu_hotkeys_#{bug}"
+
+    yast_ncurses = "#{__dir__}/yast_ncurses"
+    example_dir = "/usr/share/doc/packages/yast2-ycp-ui-bindings/examples"
+    @tui = TmuxTui.new_session "#{yast_ncurses} #{example_dir}/MenuBar1.rb"
```

Comment:
I expect this whole creation need to be redone in each test. So why not simply have helper in TmuxTui and have something like `@tui = TmuxTui.testing_client("MenuBar1.rb")` instead of those three lines?